diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x libtool.m4 -x ltoptions.m4 -x ltsugar.m4 -x ltversion.m4 -x 'lt~obsolete.m4' -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x config.rpath -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/nant-0.85/Makefile src/nant-0.85/Makefile
--- origsrc/nant-0.85/Makefile	2005-12-11 02:47:54.000000000 -0600
+++ src/nant-0.85/Makefile	2008-04-23 09:31:37.671875000 -0500
@@ -34,7 +34,7 @@
 all: bootstrap build-nant
 
 build-nant: 
-	$(NANT) -f:NAnt.build build
+	$(NANT) -t:mono-2.0 -f:NAnt.build build
 
 clean:
 	rm -fR build bootstrap
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x libtool.m4 -x ltoptions.m4 -x ltsugar.m4 -x ltversion.m4 -x 'lt~obsolete.m4' -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x config.rpath -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/nant-0.85/src/NAnt.DotNet/NAnt.DotNet.build src/nant-0.85/src/NAnt.DotNet/NAnt.DotNet.build
--- origsrc/nant-0.85/src/NAnt.DotNet/NAnt.DotNet.build	2006-03-18 00:37:34.000000000 -0600
+++ src/nant-0.85/src/NAnt.DotNet/NAnt.DotNet.build	2008-04-23 09:35:10.921875000 -0500
@@ -11,6 +11,7 @@
         <csc target="library" define="${current.build.defines}" warnaserror="true" debug="${build.debug}" output="${build.dir}/bin/${project::get-name()}Tasks.dll" doc="${build.dir}/bin/${project::get-name()}Tasks.xml">
             <nowarn>
                 <!-- do not report deprecation warnings -->
+                <warning number="0612" />
                 <warning number="0618" />
             </nowarn>
             <sources>
