--- origsrc/nant-0.85/Makefile	2005-12-11 02:47:54.000000000 -0600
+++ src/nant-0.85/Makefile	2008-04-23 09:31:37.671875000 -0500
@@ -34,7 +34,7 @@
 all: bootstrap build-nant
 
 build-nant: 
-	$(NANT) -f:NAnt.build build
+	$(NANT) -t:mono-2.0 -f:NAnt.build build
 
 clean:
 	rm -fR build bootstrap
--- origsrc/nant-0.85/src/NAnt.DotNet/NAnt.DotNet.build	2006-03-18 00:37:34.000000000 -0600
+++ src/nant-0.85/src/NAnt.DotNet/NAnt.DotNet.build	2008-04-23 09:35:10.921875000 -0500
@@ -11,6 +11,7 @@
         <csc target="library" define="${current.build.defines}" warnaserror="true" debug="${build.debug}" output="${build.dir}/bin/${project::get-name()}Tasks.dll" doc="${build.dir}/bin/${project::get-name()}Tasks.xml">
             <nowarn>
                 <!-- do not report deprecation warnings -->
+                <warning number="0612" />
                 <warning number="0618" />
             </nowarn>
             <sources>
